extensions/landing-zone/variables.tf

```diff
@@ -0,0 +1,58 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "APIkey associated with the account to use"
+  sensitive   = true
+}
+
+variable "region" {
+  type        = string
+  description = "Region to provision all resources created by this solution"
+  default     = "au-syd"
+}
+
+variable "cluster_id" {
+  type        = string
+  description = "Id of the target IBM Cloud OpenShift Cluster."
+  nullable    = false
+}
+
+variable "operator_helm_release_namespace" {
+  type        = string
+  description = "Namespace to deploy the helm releases. Default to ibm-mq-operator."
+  default     = "ibm-mq-operator"
+}
+
+variable "add_ibm_operator_catalog" {
+  type        = bool
+  description = "Flag to configure the IBM Operator Catalog in the cluster before installing the IBM MQ Operator. Default is true."
+  default     = true
+}
+
+variable "create_ibm_mq_operator_namespace" {
+  type        = bool
+  description = "Flag to create the namespace where to deploy the IBM MQ Operator. Default to false."
+  default     = false
+}
+
+variable "ibm_mq_operator_namespace" {
+  type        = string
+  description = "Namespace to install the IBM MQ operator. Default to openshift-operators."
+  default     = "openshift-operators"
+}
+
+variable "ibm_mq_operator_target_namespace" {
+  type        = string
+  description = "Namespace to be watched by the IBM MQ Operator. Default to null (operator to watch all namespaces)."
```

Comment:
`Namespace that the IBM MQ operator watches. Default is `null`, which means that the operator watches all the namespaces.`